### ISSUE TITLE
Name property for list item of Combo box 'Package manager' window is not defined properly

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#369]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4.Under "File" menu select "Add packages" or select 'Add package' button at the bottom of the workbook page.
5. Navigate to 'package Source' combo box and expand it
5. Navigate through list item and observe Inspect property

### Expected Behavior
Name property should be properly defined for list item(For e.g -'Nuget.org' )
### Actual Behavior
Name property for list item is not defined and currently is "'Xamarin.Interactive.NuGet.PackageSourceViewModel'"

Narrator Behavior: Narrator is announcing as '''Xamarin.Interactive.NuGet.PackageSourceViewModel, 1 of 2 Selected'''
NVDA behavior:NVDA is announcing as ''Xamarin.Interactive.NuGet.PackageSourceViewModel, list item focus not Selected'

### User Impact:
Users who rely on screenreader will be unable to know the Name of list item
### Screen-Shots: 
![package_source_list_item_inspect](https://user-images.githubusercontent.com/22956130/40725625-83f568fc-6441-11e8-92c6-6781be232515.png)
![package_source_list_item_narrator](https://user-images.githubusercontent.com/22956130/40725626-84238c0a-6441-11e8-99bc-e98a3a288b52.png)
![package_source_list_item_nvda](https://user-images.githubusercontent.com/22956130/40725627-844e3df6-6441-11e8-8d52-2ac81db54de3.png)

### Environment: 
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
